support/SubmitScripts/Urania.sh

```diff
@@ -0,0 +1,39 @@
+{% extends "SubmitTemplateBase.sh" %}
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Uranina -- HPC cluster of ACR division of MPI for Grav Physics, housed at the
+# Max Planck Computing & Data Facility.
+# https://docs.mpcdf.mpg.de/doc/computing/clusters/systems/Gravitational_Physics_ACR.html
+
+{% block head %}
+{{ super() -}}
+#SBATCH --nodes {{ num_nodes | default(1) }}
+#SBATCH --ntasks-per-node=1
+#SBATCH --ntasks-per-core=1
+#SBATCH --cpus-per-task=72
+#SBATCH -t {{ time_limit | default("1-00:00:00") }}
+#SBATCH -p {{ queue | default("p.urania") }}
+{% endblock %}
+
+{% block charm_ppn %}
+# Two thread for communication
+CHARM_PPN=$(expr ${SLURM_CPUS_PER_TASK} - 2)
+{% endblock %}
+
+{% block list_modules %}
+# Load compiler and MPI modules with explicit version specifications,
+# consistently with the versions used to build the executable.
+source ${SPECTRE_HOME}/support/Environments/urania.sh
+spectre_load_modules
+spectre_setup_charm_paths
```

Comment:
[optional] Might also want to `module list` after this so you can see what's loaded